src/sagemaker/image_uris.py

```diff
@@ -155,22 +155,14 @@ def retrieve(
 
     if is_jumpstart_model_input(model_id, model_version):
```

Comment:
so now if a customer passes a `model_id` AND a `framework`, `version`, `py_version`, the `framework`/`version`/`py_version` will be ignored, correct?

That's a change of behavior: is it backward incompatible?

And if yes, is there any way to mitigate the impact?

Reply:
Interesting point. Right now there are a lot of arguments that may be ignored depending on the combination of other arguments. 

Perhaps, to avoid confusion, we can emit a log indicating that the image uri won't be based on the provided framework, version, etc. if a JS model id is provided?